Resources/Prototypes/Reagents/fun.yml

```diff
@@ -0,0 +1,16 @@
+﻿- type: reagent
+  id: Carpetium
+  name: carpetium
+  desc: A mystical chemical, usually outsourced from the Clown Planet, that spawns carpets wherever it interacts with a tile.
```

Comment:
Maybe rephrase this a bit, the latter sounds kinda coder-y
"A mystical chemical, usually outsourced from the Clown Planet, that covers everything it touches in carpet".